src/common/slate_serializers/to_rtf.js

```diff
@@ -91,15 +99,15 @@ const leafInElement = (node) => {
 }
 
 const makeList = (node, doc) => {
-  const items = node.children.map(makeListItem).join(' ')
-
-  const prefix = "\\ls1\\ilvl0\\cf2 \\kerning1\\expnd0\\expndtw0 \\outl0\\strokewidth0 "
-
-  const list = rtf.Utils.makeRtfCmd(prefix, items, '', false)
-  doc.addCommand(list)
+  const items = node.children.map(makeListItem).join('\n')
+  doc.addCommand(items)
 }
 
+// There seems to be one thing about these lists that is weird
+// The first list item has some extra spacing between the left margin
+// and the bullet point. The rest of the items do not have this and I
+// can't figure out why it is there. (Matt)
 const makeListItem = (node) => {
   const text = node.children.flatMap(ch => ch.text).join(' ')
-  return rtf.Utils.makeRtfCmd("{\\listtext	\\uc0\\u8226	}", text, "\\\n", false)
-}
\ No newline at end of file
+  return `{\\pard\\fi-300\\li100\\bullet\\tab ${text} \\par}`
```

Comment:
is there some documentation that you found for rtf documents? It might be valuable to keep that somewhere we can all find (maybe even just a link in a comment)

Reply:
Yeah I can add it to this file.